core/src/main/kotlin/entity/GuildScheduledEvent.kt

```diff
@@ -117,6 +117,8 @@ public class GuildScheduledEvent(
     /** The cover image hash of this event. */
     public val imageHash: String? get() = data.image.value
 
+    public val cover: Asset? get() = data.image.value?.let { Asset.GuildEventCover(guildId, it, kord) }
```

Comment:
i think `image` would be the better name - it's consistent with the already existing `imageHash`